### PR TITLE
Create roadmap-item issue template

### PR DESCRIPTION
**Pull request by vincent6767**
# Description
The context of the roadmap item can be read here: https://github.com/o3de/sig-release/issues/79.

# Action Needed
Since the RFC leverage three labels, kind/roadmap, needs-triage, and triage/accepted, please create this label in your SIG repo if it does not exist already once you merge the PR.

### needs-triage
Taken from [O3DE label](https://github.com/o3de/o3de/labels?q=needs-triage)
- label name: needs-triage
- description: Indicates an issue or PR lacks a triage/foo label and requires one

### triage/accepted
Taken from [O3DE Label](https://github.com/o3de/o3de/labels?q=triage%2Faccepted)
- label name: triage/accepted
- description: Indicates an issue or PR is ready to be actively worked on.

### kind/roadmap
Taken from [O3DE Label](https://github.com/o3de/o3de/labels?q=kind%2Froadmap)
- label name: **kind/roadmap**
- description: **Categorizes an issue that goes in the O3DE Public Roadmap**

Signed-off-by: Vincent <vincentbiasa6767@gmail.com>

### CODE REVIEW

**Review by lmbr-pip — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/roadmap-item.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: 'Roadmap Item'
+about: Create roadmap item for this project
+title: Roadmap Item =description=
+labels: kind/roadmap
```
Comment (lmbr-pip) [inline .github/ISSUE_TEMPLATE/roadmap-item.md:5]: We probably need a 'needs-triage' label so if roadmap items are created they are easy to find for review. However volume of items should be low and we can use project board to track status of such items.

Comment (vincent6767) [reply .github/ISSUE_TEMPLATE/roadmap-item.md:5]: Yes! I add the needs-triage label in [this PR](https://github.com/o3de/sig-simulation/pull/37). Will update this PR with the new changes by this week

Comment (vincent6767) [reply .github/ISSUE_TEMPLATE/roadmap-item.md]: @lmbr-pip Updated the PR. Please help review!

**Comment by vincent6767** [comment]
Thanks for approving @lmbr-pip Do I need to wait for 
@kberg-amzn 's approval before merging?

**Comment by vincent6767** [comment]
I'll proceed merging the PR.

**Pull request merged**